scripts/set-environment

```diff
@@ -76,18 +94,25 @@ echo
 # Avoid ssh prompting when connecting to new ssh hosts
 mkdir -p $HOME/.ssh && echo "StrictHostKeyChecking no" >> "$HOME/.ssh/config"
 
+# Create the artifacts directory if needed
+if [ ! -d "$ARTIFACTS_FULL_DIR" ]; then
```

Comment:
The vr job won't be calling this script directly, so we don't need to mkdir the artifacts dir here. Just define the path to it, create it when using it for vr.